packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2552,31 +2511,6 @@ export class ContainerRuntime
 		}
 	}
 
-	private async loadIdCompressor() {
-		if (
-			this._idCompressor === undefined &&
-			this.idCompressorMode !== undefined &&
-			this._loadIdCompressor === undefined
-		) {
-			this._loadIdCompressor = this.createIdCompressor()
-				.then((compressor) => {
-					// Finalize any ranges we received while the compressor was turned off.
-					const ops = this.pendingIdCompressorOps;
-					this.pendingIdCompressorOps = [];
-					for (const range of ops) {
-						compressor.finalizeCreationRange(range);
-					}
-					assert(this.pendingIdCompressorOps.length === 0, 0x976 /* No new ops added */);
-					this._idCompressor = compressor;
-				})
-				.catch((error) => {
-					this.logger.sendErrorEvent({ eventName: "IdCompressorDelayedLoad" }, error);
-					throw error;
-				});
-		}
-		return this._loadIdCompressor;
-	}
-
```

Comment:
With the new behavior, transitioning to "delayed" has no immediate effect (i.e., the IdCompressor is loaded/created on future container loads).  We rely on sufficient time elapsing to "age out" old sessions.